keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -482,6 +485,78 @@ are properly reloading tokens by:
   - upgrade test:
     test/e2e/upgrades/serviceaccount_admission_controller_migration.go
 
+### Rollout, Upgrade and Rollback Planning
+
+_This section must be completed when targeting beta graduation to a release._
+
+- **How can a rollout fail? Can it impact already running workloads?**
+  Try to be as paranoid as possible - e.g., what if some components will restart
+  mid-rollout?
+
+- **What specific metrics should inform a rollback?**
+
+- **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  Describe manual testing that was done and the outcomes.
```

Comment:
manually test rollback / disabling the feature, verify existing workloads still authenticate correctly, describe test here